src/angular-business/header/header/header.component.ts

```diff
@@ -62,22 +117,35 @@ export type SbbHeaderMenuToggleResult = 'open' | 'close';
   host: {
     class: 'sbb-header',
     '[class.sbb-header-opened]': 'this.opened',
+    '[class.sbb-header-menus-collapsed]': 'this._menusCollapsed',
   },
 })
-export class SbbHeader implements OnInit, AfterViewInit, OnDestroy {
+export class SbbHeader implements OnInit, OnChanges, AfterViewInit, OnDestroy {
   /** Main title shown in the header. */
   @Input() label: string;
 
   /** Subtitle shown below the main title, if present. */
   @Input() subtitle?: string;
 
+  /**
+   * The breakpoint on which to collapse the header menus into the burger menu.
+   * Matching the given breakpoint or anything below, will collapse the menus.
+   * (e.g. 'tablet' will collapse the menu, if 'tablet' or 'mobile' is matched)
+   * See breakpoint documentation for specific values.
+   */
+  @Input() collapseBreakpoint: SbbHeaderCollapseBreakpoint = 'tablet';
+
   /**
    * String representing the kind of environment the application is running in.
    * Will be shown in a ribbon, top-left corner of the header.
+   * @deprecated Use the sbb-header-environment element.
```

Comment:
Perhaps a little more deprecated documentation here or something like 'See documentation of sbb-header'